mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.EraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
```

Comment:
The `findByIdAndDelete` method should be awaited since it returns a promise. Add `await` before `Order.findByIdAndDelete(req.params.orderId)` to ensure the function waits for the promise to resolve before proceeding.